src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -0,0 +1,45 @@
+package mate.jdbc.model;
+
+public class Manufacturer {
```

Comment:
Tip for further improvements: you can look through Lombok annotations to decrease code size in your POJOs. For example @AllArgsConstructor, @NoArgsConstructor, etc.
POJO: https://www.baeldung.com/java-pojo-class

You can find out more information here:
https://www.baeldung.com/intro-to-project-lombok